corehq/apps/domain/models.py

```diff
@@ -548,7 +549,13 @@ def _get_by_name(stale=False):
             if not isinstance(result, Domain):
                 # A stale view may return a result with no doc if the doc has just been deleted.
                 # In this case couchdbkit just returns the raw view result as a dict
-                return None
+                try:
+                    result = Domain.get(name)
```

Comment:
Yeah, this is nonsense, it's left over from a different incarnation of this PR. Removing.